cypress/integration/learn/index.js

```diff
@@ -48,3 +48,159 @@ describe('Learn Landing page', function() {
     });
   });
 });
+
+describe('Quotes', () => {
+  beforeEach(() => {
+    cy.visit('/');
+    cy.contains("Get started (it's free)").click({ force: true });
```

Comment:
Do these need forcing?  I see this happens throughout the tests, but I wasn't sure what the reasoning was.

Reply:
Yes. These do need forcing due to the overlapping of some elements. If you try without force you'll get an error and it suggests to use force.

Reply:
Ah, I see that now.  Any idea if that's a Cypress bug or something wrong on our end?  I'm leaning towards the former, but I'm not 100% sure.

If you think it's an issue with Cypress, could you add a comment explaining that?  If they fix it, we'll want to remove the force.

Reply:
I think it's a Cypress specific problem. I did some digging and this is what I found -

Cypress scrolls the page as it loads and when we want to click a block/superblock, the banner covers the it. I tried using `.scrollIntoView()` with and without an offset but it doesn't seem to work. (I'm leaning towards this being a bug)

Reply:
You were spot on about the scrolling.  What's happening is that Cypress tries to scroll the button into view, but it doesn't take account of the nav, because it's absolutely positioned.  If we use `position: fixed` it should just work.

@ahmadabdolsaheb can we just change to `fixed`?